plugins/feeds/public/hybrid_analysis.py

```diff
@@ -32,7 +32,7 @@ def analyze(self, item):
         context = {'source': self.name}
 
         if 'vxfamily' in item:
-            tags.append(' '.join(item['vxfamily'].split('.')))
+            context['vxfamily'] = ' '.join(item['vxfamily'].split('.'))
```

Comment:
This is a bit convoluted :P Why not just do `context['vxfamily'] = item['vxfamily']`?

Reply:
becaux item['vxfamiliy'] is a list